react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader 3.0.0-beta.6
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react"
+
+declare module "react-hot-loader" {
```

Comment:
Should be safe to remove this wrapper and just use the file itself as the module definition.